custom_components/tahoma/tahoma_entity.py

```diff
@@ -199,14 +199,6 @@ async def async_cancel_command(self, exec_id: str):
         """Cancel device command in async context."""
         await self.coordinator.client.cancel_command(exec_id)
 
-    def get_base_device_url(self):
```

Comment:
Why is this removed?

Reply:
I didn’t know what name gave to my function as I want to return the base url and the index. :S

Reply:
`split_device_url`? Don't know either..

Reply:
Ahah you see :D